registration/app/registration/services/NotificationRegistrar.scala

```diff
@@ -14,8 +15,31 @@ object RegistrationResponse {
   implicit val jf = Json.format[RegistrationResponse]
 }
 
+case class StoredRegistration(deviceId: String, platform: Platform, userId: Option[UniqueDeviceIdentifier], tagIds: Set[String], topics: Set[Topic])
+
+object StoredRegistration {
+  import play.api.libs.json._
+
+  implicit val jf = Json.format[StoredRegistration]
+
+  def fromRegistration(registration: Registration) = {
+    StoredRegistration(
+      deviceId = registration.deviceId,
+      platform = registration.platform,
+      userId = Some(registration.udid),
+      tagIds = registration.topics.map(_.id),
```

Comment:
Do you think we need tagIds and topics? We can always go from the list topics to the list of topics Id.